frontend/express/public/core/events/javascripts/countly.overview.views.js

```diff
@@ -353,7 +370,7 @@
             return {
                 description: CV.i18n('events.overview.title.new'),
                 disabledDatePicker: '1months',
-                monitorEventsLegend: {"show": false},
+                monitorEventsLegend: { "show": false },
```

Comment:
Let's try to sync on formatting styles and use eslint in the repo and not do unnecessary changes

Reply:
hmm, strange I do run eslinteslint . --ext .js --fix before commit, and it even passed eslint check of github.
will change it back nonetheless